src/GetterXcApi.js

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
Find a better naming xcomponentConfiguration
and a method getXmlConfiguration
easier to understand and ready for the next steps (get xcapi xml from server)
Create a sub folder configuration